suite/runsuite_common_pre.cmake

```diff
@@ -1,5 +1,5 @@
 # **********************************************************
```

Comment:
It'd be useful to add to the PR description the list of our workflows that do have the required cmake version already.

Reply:
Done.  (It's all of them.)